cpp/pylib_test.cc

```diff
@@ -26,6 +26,12 @@ TEST os_path_test() {
   PASS();
 }
 
+TEST isdir_test() {
+  ASSERT(path_stat::isdir(StrFromC(".")));
+  ASSERT(path_stat::isdir(StrFromC("/")));
```

Comment:
I sometimes add stuff like `!isdir("/nonexistent_ZZZ")` , which is very cheesy, but I think acceptable ...

We do run our tests in containers except on local machines ...